TezosSwift/Client/TezosError.swift

```diff
@@ -89,6 +91,18 @@ public enum TezosError: Error {
     case parameterError(reason: ParameterReason)
     /// Error when decoding Tezos response data
     case decryptionFailed(reason: DecryptionReason)
-    ///Error when encrypting data 
+    /// Error when encrypting data
     case encryptionFailed(reason: Error)
+	/// User cancel
+	case userCancel
```

Comment:
I'd maybe just go with `case canceled`? We could cancel the operation for other reasons than user canceling the operation, I think.